.mvn/wrapper/maven-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 #
-# Copyright HomeAway, Inc 2018-Present. All Rights Reserved.
+# Copyright Expedia, Inc 2018-2019. All Rights Reserved.
```

Comment:
Sorry but this actually needs to be like this:

`Copyright (C) 2018-2019 Expedia, Inc.`

We should definitely do a stripped down eg-oss-parent POM which has all this stuff in it but for now maybe a small new PR with this change ^?

Reply:
Yes, yes. Please submit new PR with this change. 
Good catch @massdosage